### PR TITLE
Upgraded sp-action-group to v0.37.0 and added theme/scale support in test plugin

### PR DESCRIPTION
**Pull request by inasirhussain**
Closes - https://jira.corp.adobe.com/browse/UXP-22038

### CODE REVIEW

File: projects/swc-starter-webpack/src/index.html [hunk 1]
```diff
@@ -67,291 +67,379 @@
...
-            </div>
-
-            <div id="sp-action-button" class="tabcontent">
-                <!--#include sp-action-button.html -->
-            </div>
-
-            <div id="sp-action-group" class="tabcontent">
-                <!--#include sp-action-group.html -->
-            </div>
+        <sp-theme
```
[the remark this answers is not in the document]
Comment (inasirhussain) [reply projects/swc-starter-webpack/src/index.html:70]: Yes, This PR contains primarily upgrade of sp-action-group as a control and rest changes are in swc-starter-webpack plugin

File: projects/swc-starter-webpack/src/attach-event-helper.js [hunk 2]
```diff
@@ -42,3 +42,23 @@ function attachEvents(tabName) {
...
+    var value = selectObject.value;
+    document.querySelector('#theme-block').setAttribute('color', value);
+    var globalDiv = document.getElementById('global');
+    if (
+        value == 'dark' ||
+        value == 'darkest' ||
+        value == 'light' ||
+        value == 'lightest'
+    ) {
+        globalDiv.style.backgroundColor =
```
Comment (deepthi-majji) [inline projects/swc-starter-webpack/src/attach-event-helper.js:56]: We are setting the same background color for all themes?

File: projects/swc-starter-webpack/src/index.js [hunk 3]
```diff
@@ -32,10 +32,10 @@ import '@spectrum-web-components/popover/sp-popover.js';
 import '@spectrum-web-components/tooltip/sp-tooltip.js';
 import '@spectrum-web-components/picker-button/sp-picker-button.js';
 import '@spectrum-web-components/help-text/sp-help-text.js';
-import '@spectrum-web-components/menu/sp-menu.js';
-import '@spectrum-web-components/menu/sp-menu-item.js';
-import '@spectrum-web-components/menu/sp-menu-group.js';
-import '@spectrum-web-components/menu/sp-menu-divider.js';
+// import '@spectrum-web-components/menu/sp-menu.js';
+// import '@spectrum-web-components/menu/sp-menu-item.js';
+// import '@spectrum-web-components/menu/sp-menu-group.js';
```
Comment (deepthi-majji) [inline projects/swc-starter-webpack/src/index.js:37]: Can we remove them instead of commenting? Or we are saving it future PR?

Comment (inasirhussain) [reply projects/swc-starter-webpack/src/index.js:37]: I have added them back, thought to save for future PR. Its fine to keep for now

File: projects/swc-starter-webpack/src/attach-event-helper.js (hunk 2 above)
Comment (inasirhussain) [reply projects/swc-starter-webpack/src/attach-event-helper.js:56]: It's a variable which takes appropriate color based on the theme selected, similar variable is being used in SWC storybook as well to provide coordinated background for the corresponding themes. I will change the implementation a bit applying this color variable directly over the div id#global

**Review by deepthi-majji — APPROVED** [review]

**Pull request merged**